app.Makefile

```diff
@@ -49,10 +50,13 @@ app/build:: ;
 app/install:: app/build \
               .build/var/MARKETPLACE_TOOLS_PATH \
               .build/var/APP_DEPLOYER_IMAGE \
-              .build/var/APP_PARAMETERS
-	$(MARKETPLACE_TOOLS_PATH)/scripts/start.sh \
+              .build/var/APP_PARAMETERS \
+              .build/marketplace/dev
+	$(call print_target)
+	"$(MARKETPLACE_TOOLS_PATH)/scripts/start.sh" \
 	    --deployer='$(APP_DEPLOYER_IMAGE)' \
-	    --parameters='$(APP_PARAMETERS)'
+	    --parameters='$(APP_PARAMETERS)' \
+	    --entrypoint='/bin/deploy.sh'
```

Comment:
can we remove this and make it have a default?

Reply:
Done.